### PR TITLE
Add Math::BigInt and Math::BigFloat as recommended dependences

### PR DESCRIPTION
**Pull request by pali**

### CODE REVIEW

File: Makefile.PL [hunk 1]
```diff
@@ -125,9 +133,9 @@ WriteMakefile(
       resources =>
       {
         license     => 'http://dev.perl.org/licenses/',
```
Comment (Grinnz) [inline Makefile.PL]: The license resource is an array in meta-spec 2.

File: Makefile.PL [hunk 2]
```diff
@@ -125,9 +133,9 @@ WriteMakefile(
...
-        bugtracker  => 'https://github.com/rurban/Cpanel-JSON-XS/issues',
+        bugtracker  => { web => 'https://github.com/rurban/Cpanel-JSON-XS/issues' },
         # Note: https://rt.cpan.org/Public/Dist/Display.html?Queue=Cpanel-JSON-XS is also observed
-        repository  => 'https://github.com/rurban/Cpanel-JSON-XS',
+        repository  => { url => 'https://github.com/rurban/Cpanel-JSON-XS' },
```
Comment (Grinnz) [inline Makefile.PL]: This should have at the least `type => 'git'`.

File: Makefile.PL (hunk 1 above)
Comment (pali) [reply Makefile.PL]: fixed

File: Makefile.PL (hunk 2 above)
Comment (pali) [reply Makefile.PL]: fixed

**Comment by rurban** [comment]
LGTM

**Pull request merged**